docs/data/date-pickers/shortcuts/CustomizedRangeShortcuts.tsx

```diff
@@ -85,12 +101,20 @@ function CustomRangeShortcuts(props: PickersShortcutsProps<DateRange<Dayjs>>) {
         dense
         sx={(theme) => ({
           display: 'flex',
+          maxWidth: '625px',
```

Comment:
Could you document why 625? 
And potentially use a constant instead of hardcoding the value.